common/src/main/java/org/apache/rocketmq/common/ControllerConfig.java

```diff
@@ -59,6 +59,11 @@ public class ControllerConfig {
      */
     private volatile boolean notifyBrokerRoleChanged = true;
 
+    /**
+     * How many DLedger logs are committed to trigger a snapshot
+     */
+    private int statemachineSnapshotThreshold = 1000;
```

Comment:
There's also a parameter called "maxSnapshotReservedNum", which is for setting the max number of snapshot files keeping within the store. I believe it should be added to the controller config as well.

Reply:
Sure